### ISSUE TITLE
Adjustable camera PTZ speeds

### ISSUE THREAD

**Issue by jamesremuscat** [staldates/av-control#20]
(Deferred from HD upgrade work)

It would be nice to have a way of adjusting the pan, tilt and zoom speeds of the camera.

**Comment by rjmunro** [comment]
For pan/tilt, you could have a square on the touch screen with a point in the middle and as you move further from the point, the speed increases in that direction.

**Comment by jamesremuscat** [comment]
The easiest way might be to complete the work on using a joystick to control the cameras - it's pretty close, and I should have time to get back to that now other projects have concluded.

**Comment by jamesremuscat** [comment]
Now we have joystick control with (after #33) configurable sensitivity, I don't see any value in further work in this area, so I'll close this now.
